plugins/modules/swim_workflow_manager.py

```diff
@@ -1069,14 +1267,23 @@ def get_diff_import(self):
                 self.result['changed'] = False
                 return self
 
+            self.log("image_type - {0}".format(import_type))
             if import_type == "remote":
-                image_name = self.want.get("url_import_details").get("payload")[0].get("source_url")
-            else:
+                image_names = self.want.get("url_import_details").get("payload")[0].get("source_url")
+                self.log(image_names)
```

Comment:
Write the complete log messages otherwise how will we know that these are images names.
```
self.log("Image(s) '{0}' to be imported in Cisco Catalyst Center".format(image_names), "INFO")

Reply:
updated the code